src/main/resources/db/migration/v0.35.0.9__add_htp_tags.sql

```diff
@@ -1,218 +1,129 @@
-INSERT INTO vocabularyterm (id, name, definition, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'age', 'study that uses samples of the same developmental stage but >1 age, especially postnatal studies',id FROM vocabulary WHERE vocabularylabel = 'data_set _category_tags';
-INSERT INTO vocabularyterm_synonyms (vocabularyterm_id, synonyms) SELECT id,'' FROM vocabularyterm WHERE name = 'age';
-INSERT INTO vocabularyterm (id, name, definition, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'aging', 'A developmental process that is a deterioration and loss of function over time. Aging includes loss of functions such as resistance to disease, homeostasis, and fertility, as well as wear and tear. Aging includes cellular senescence, but is more inclusive. May precede death and may succeed developmental maturation (GO:0021700).',id FROM vocabulary WHERE vocabularylabel = 'data_set _category_tags';
-INSERT INTO vocabularyterm_synonyms (vocabularyterm_id, synonyms) SELECT id,'' FROM vocabularyterm WHERE name = 'aging';
-INSERT INTO vocabularyterm (id, name, definition, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'cell aging', 'Progression of a cell from its inception to the end of its lifespan, including replicative aging (number of cell divisions a cell undergoes before dying) and chronological aging (number of days a culture remains viable in stationary phase).',id FROM vocabulary WHERE vocabularylabel = 'data_set _category_tags';
-INSERT INTO vocabularyterm_synonyms (vocabularyterm_id, synonyms) SELECT id,'' FROM vocabularyterm WHERE name = 'cell aging';
-INSERT INTO vocabularyterm (id, name, definition, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'amino acid metabolism', 'The cellular reactions and pathways used in the biosynthesis and catabolism of amino acids.',id FROM vocabulary WHERE vocabularylabel = 'data_set _category_tags';
-INSERT INTO vocabularyterm_synonyms (vocabularyterm_id, synonyms) SELECT id,'' FROM vocabularyterm WHERE name = 'amino acid metabolism';
-INSERT INTO vocabularyterm (id, name, definition, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'amino acid utilization', 'Utilization of different amino acids, combinations of amino acids, or altered levels of amino acids (e.g. amino acid limitation) during nutritional shifts.',id FROM vocabulary WHERE vocabularylabel = 'data_set _category_tags';
-INSERT INTO vocabularyterm_synonyms (vocabularyterm_id, synonyms) SELECT id,'' FROM vocabularyterm WHERE name = 'amino acid utilization';
-INSERT INTO vocabularyterm (id, name, definition, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'anatomical structure', 'study that characterizes expression in one or more tissues',id FROM vocabulary WHERE vocabularylabel = 'data_set _category_tags';
+UPDATE vocabulary SET vocabularylabel = 'data_set_category_tags' WHERE vocabularylabel = 'data_set _category_tags';
```

Comment:
How was this vocabulary created in the first place?  As far as I can tell it was manually created on alpha rather than in a migration.  If so, then this migration will fail when deployed to beta/production as the vocabulary that you are inserting terms for won't exist (unless created ahead of time).  It would be better to delete the vocabulary on alpha and recreate it here in the migration - that way no manual work is required prior to deployment to beta/production.

Reply:
Yes, It was created manually on alpha. I will delete it and create again in migration file. Thank you. I didnt think of this scenario.